uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -609,14 +609,24 @@ public synchronized void updateCurrentStatus() {
     return result;
   }
 
+  private String getHostname(String instanceId) {
```

Comment:
Can we factor out this function with line https://github.com/uber/uReplicator/blob/master/uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java#L265

Reply:
this function is for one time get host name,  L265 need to retrieve hostname from instanceIdAndNameMap multiple times, IMO, it's ok to have separate implementation.